src/docker-osx-dev

```diff
@@ -379,13 +384,63 @@ function init_boot2docker {
     log_info "Starting Boot2Docker VM"
     boot2docker start --vbox-share=disable
   fi
+
+  configure_boot2docker_vars
+}
+
+################################################################################
+# docker-machine manipulation
+################################################################################
+
+function get_json_val () { 
```

Comment:
This function needs a better name and comments explaining how it is intended to be used. As it is, I can't tell what JSON it's reading (from a file or stdin or as a param?) nor what "val" it's trying to get (a single key or the whole JSON blob?).